.github/workflows/test.yml

```diff
@@ -0,0 +1,80 @@
+name: CI - Test package
+
+on:
+  push:
+    branches: [main]
+  pull_request:
+    branches: [main]
+    types: [opened, reopened, synchronize, ready_for_review]
+
+
+jobs:
+  lint:
+    name: Lint
+    runs-on: ubuntu-latest
+    if: ${{ github.event_name == 'push' || !github.event.pull_request.draft }}
```

Comment:
```suggestion
    if: github.event.pull_request.draft == false
```

Reply:
`on: push` will have no effect then.
It *might* be useful to have the CI running on eventual pushes on main (agreed in a normal workflow one must do a PR)

Reply:
> on: push will have no effect then.

No, it will work anyway. (tested)

Reply:
> on: push will have no effect then.

No, it works even with pushs on main.